web/telemetry/publisher.go

```diff
@@ -0,0 +1,50 @@
+package telemetry
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+
+	"github.com/google/uuid"
+)
+
+type TelemetryPublisher struct {
+	apiHost    string
+	httpClient *http.Client
+}
+
+func (tp *TelemetryPublisher) Publish(telemetryName string, installationID uuid.UUID, extractedTelemetry interface{}) error {
+	endpoint := fmt.Sprintf("%s/api/collect/hosts", tp.apiHost)
```

Comment:
I guess we could already pass this `hosts` endpoint part as variable.
Anyway, I see many questions just below. We can refine on the road

Reply:
Yep, we'll followup on those, thanks :wink: